SensorsAnalyticsSDK/src/main/java/com/sensorsdata/analytics/javasdk/HelloSensorsAnalytics.java

```diff
@@ -13,7 +13,7 @@ public static void main(String[] args) throws Exception {
 
     // DebugConsumer
     final SensorsAnalytics sa = new SensorsAnalytics(new SensorsAnalytics.DebugConsumer
-        (SA_SERVER_URL, true));
+        (SA_SERVER_URL, true, null));
```

Comment:
Do we need to update the example with an explicit null if the previous behaviour still works?

Reply:
Good point.